content/oci/oci.go

```diff
@@ -70,16 +70,25 @@ func New(root string) (*Store, error) {
 
 // NewWithContext creates a new OCI store.
 func NewWithContext(ctx context.Context, root string) (*Store, error) {
+	rootAbs, err := filepath.Abs(root)
+	if err != nil {
+		return nil, fmt.Errorf("failed to resolve absolute path for %s: %w", root, err)
+	}
```

Comment:
This check is unnecessary since `NewStorage()` has already done the check.

Reply:
This is still needed because `indexPath` relies on the root path. See line 85 and line 91.